code/modules/clothing/clothing.dm

```diff
@@ -47,6 +47,10 @@
 			var/mob/living/carbon/human/H = loc
 			H.update_inv_by_slot(slot_flags)
 		return 1
+	if(istype(I, /obj/item/weapon/screwdriver))
```

Comment:
Prefer `I.is_screwdriver(user)` for the new proc that makes coins act as screwdrivers.

Reply:
Can't figure out how to get that to work with this.

Reply:
I.is_screwdriver(user) will return 1 if its a screwdriver, so replace `if(istype(I, /obj/item/weapon/screwdriver))` with `if(I.is_screwdriver(user))`

the user param is relevant for cases where the obj would only be considered a screwdriver when on harm intent(or other user related things) eg. coins

Reply:
Yeah, sadly, I doesn't support `is_screwdriver(user)` in this case. It is a proc that does not exist.

Reply:
i forgot to update my code and exxion told me to update my code and now the proc works thank you and goodbye

Reply:
u a silli boi